src/components/comparison/layers-panel/list-item-wrapper.tsx

```diff
@@ -3,7 +3,7 @@ import styled, { css, useTheme } from "styled-components";
 import { ExpandState } from "../../../types";
 import { color_brand_quaternary } from "../../../constants/colors";
 
-import ChevronIcon from "../../../../public/icons/chevron.svg?svgr";
+import { ReactComponent as ChevronIcon}  from "../../../../public/icons/chevron.svg?svgr";
```

Comment:
It doesn't look correct. The config is set up on default export. There is a warning:
```
WARNING in ./src/components/comparison/layers-panel/list-item-wrapper.tsx 81:483-494
export 'ReactComponent' (imported as 'ChevronIcon') was not found in '../../../../public/icons/chevron.svg?svgr' (possible exports: default)
```